servicenow/servicenow.js

```diff
@@ -0,0 +1,164 @@
+//Copyright 2020 Google LLC
+//Licensed under the Apache License, Version 2.0 (the "License");
+//you may not use this file except in compliance with the License.
+//You may obtain a copy of the License at
+//    https://www.apache.org/licenses/LICENSE-2.0
+//Unless required by applicable law or agreed to in writing, software
+//distributed under the License is distributed on an "AS IS" BASIS,
+//WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//See the License for the specific language governing permissions and
+//limitations under the License.
+
+
+(function process(/*RESTAPIRequest*/ request, /*RESTAPIResponse*/ response) {        
+
+
+        var writer = response.getStreamWriter();
+
+
+        hdrs = {};
+
+
+                var x=request.body.data;
+                var inc='';
```

Comment:
A lot of variables defined at here are not used in the file, you may consider to clean them up.

Reply:
let's leave for now...the code works and I don't want to accidentally break anything.

Reply:
You can just do a search and replace, if they are not used in anywhere, we can safely remove it. This is the open source project, everybody can read it, we probably don't want to confuse them. Is that possible you can do another round of testing after the cleanup to make sure it is working?

Reply:
removed extra variables